tests/dockerfile_test.py

```diff
@@ -43,31 +43,31 @@ def test_parse_string_success():
         dockerfile.Command(
             cmd='from', sub_cmd=None, json=False, flags=(),
             value=('ubuntu:xenial',),
-            start_line=1, original='FROM ubuntu:xenial',
+            start_line=1, end_line=1, original='FROM ubuntu:xenial',
         ),
         dockerfile.Command(
             cmd='run', sub_cmd=None, json=False, flags=(),
             value=('echo hi > /etc/hi.conf',),
-            start_line=2, original='RUN echo hi > /etc/hi.conf',
+            start_line=2, end_line=2, original='RUN echo hi > /etc/hi.conf',
         ),
         dockerfile.Command(
             cmd='cmd', sub_cmd=None, json=True, flags=(), value=('echo',),
-            start_line=3, original='CMD ["echo"]',
+            start_line=3, end_line=3, original='CMD ["echo"]',
         ),
         dockerfile.Command(
             cmd='healthcheck', sub_cmd=None, json=False,
             flags=('--retries=5',), value=('CMD', 'echo hi'),
-            start_line=4, original='HEALTHCHECK --retries=5 CMD echo hi',
+            start_line=4, end_line=4, original='HEALTHCHECK --retries=5 CMD echo hi',
```

Comment:
looks like the linters are pointing out that this line got too long, an easy fix is to put a linebreak before `original` 👍